app/api/sessions.py

```diff
@@ -148,6 +150,11 @@ def before_update_object(self, session, data, view_kwargs):
         if session.is_locked and data.get('is_locked') == session.is_locked:
             raise ForbiddenException({'source': '/data/attributes/is-locked'}, "Locked sessions cannot be edited")
 
+        speakers_call = safe_query(self, SpeakersCall, 'event_id', session.event_id, 'event-id')
+        if speakers_call.ends_at.replace(tzinfo=None) <= datetime.now().replace(tzinfo=None):
```

Comment:
Now I won't be able to delete the session as well.

Second, you have made it timezone unaware

Reply:
@iamareebjamal - Shall the sessions be deleted after the CFS is ended?

Reply:
Admin/Co-organizer should be able to edit and delete both